config.yml

```
@@ -0,0 +1,32 @@
+  - Veggies
+  - HeavyAlcoholConsump
+  - AnyHealthcare
+  - NoDocbcCost
+  - GenHlth
+  - DiffWalk
+  - Sex
+  - Age
+  - Education
+  - Income
+
+target: Diabetes_012
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)